Makefile

```diff
@@ -1,7 +1,13 @@
-TAG ?= v0.24.1
+# TO EDIT
+TAG ?= v0.24.2
+GH_USERNAME ?= jorisvandenbossche
+
 PANDAS_VERSION=$(TAG:v%=%)
+PANDAS_BASE_VERSION=0.24
```

Comment:
I didn't find a way to automatically convert something like "0.24.2" into "0.24" with make, so for now just hardcoded it here (so it should be in the section "to edit"). 
Anybody an idea how to do this automatically?

Reply:
Pushed an awk version.